<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProMsgWidget</class>
 <widget class="QWidget" name="ProMsgWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>项目成员</string>
  </property>
  <property name="windowIcon">
   <iconset resource="SIMS_res.qrc">
    <normaloff>:/SIM_System_Pics/resources/icon_login_2.png</normaloff>:/SIM_System_Pics/resources/icon_login_2.png</iconset>
  </property>
  <widget class="QTableWidget" name="ApplyWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>461</width>
     <height>192</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>152</number>
   </attribute>
   <column>
    <property name="text">
     <string>学生编号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>学生姓名</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>申请理由</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;申请人&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;项目成员&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="AdmittedWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>461</width>
     <height>192</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>152</number>
   </attribute>
   <column>
    <property name="text">
     <string>学生编号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>学生姓名</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>申请理由</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;项目名称&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="NameEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>361</width>
     <height>23</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>461</width>
     <height>40</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QToolButton" name="FlushButton">
      <property name="text">
       <string>刷新</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="ConfirmButton">
      <property name="text">
       <string>通过申请</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="QuitButton">
      <property name="text">
       <string>拒绝申请</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>NameEdit</tabstop>
  <tabstop>ApplyWidget</tabstop>
  <tabstop>AdmittedWidget</tabstop>
 </tabstops>
 <resources>
  <include location="SIMS_res.qrc"/>
 </resources>
 <connections/>
</ui>
